UniMath/CategoryTheory/.package/files

```diff
@@ -18,13 +18,13 @@ limits/initial.v
 limits/graphs/initial.v
 limits/zero.v
 limits/graphs/zero.v
+limits/bincoproducts.v
 limits/coproducts.v
-limits/arbitrary_coproducts.v
-limits/graphs/coproducts.v
+limits/graphs/bincoproducts.v
```

Comment:
:-)

I was surprised to see "the standard" in a comment, as though my proposal had been approved or even acclaimed.